### DISCUSSION TITLE
Does increasing maximum controlnets negatively affect anything in anyway?

### DISCUSSION THREAD

**Discussion by Rootybooty** [Mikubill/sd-webui-controlnet#1604] [Q&A]
Allocating and reserving more memory for example is one thing that I don't want. If there's no reason not to though then I may just go forward and increase it to 10 if I ever do want to use up to that many.

**Comment by huchenlei** [comment]
I have seem people have 10 or even more ControlNet units enabled. (More requires changing ControlNet code). 

Prior `1.1.224`, an openpose editor iframe will be loaded for each units (Around 0.5MB memory for each iframe), but in `1.1.224` we have made the editor iframe to lazy load, so this won't be an issue.

**Comment by CapsAdmin** [comment]
I find that adding more controlnet modules will slow down the time it takes from when you click generate to when the inference starts. See https://github.com/Mikubill/sd-webui-controlnet/issues/1461

This won't be recorded in how long inference takes, because the recording starts after this delay.
